models/articles.models.js

```diff
@@ -0,0 +1,13 @@
+const db = require("../db/connection")
+
+exports.selectArticlesById = (articleId) => {
+    return db.query(`
+    SELECT * 
+    FROM articles WHERE article_id = $1`, [articleId])
+    .then(({rows}) => {
+        if(!rows.length){
+            return Promise.reject({status: 404, msg: "topic not found with this article id"})
+        }        
```

Comment:
The reject message isn't correct we're not dealing with topics here?
But also take a look at the PSQL error code that comes back when we input either a wrong data type or an article that doesn't exist as these would be a better way to deal with the errors.